packages/compiler-vapor/__tests__/compile.test.ts

```diff
@@ -176,6 +176,18 @@ describe('compile', () => {
         expect(code).toMatchSnapshot()
         expect(code).not.contains('effect')
       })
+
+      test('on nested plain element', async () => {
+        const code = await compile(`<div><div :id="foo" v-once /></div>`)
+        expect(code).toMatchSnapshot()
+        expect(code).not.contains('effect')
+      })
```

Comment:
This looks very similar to the test I added earlier, which I had previously marked as a failure.

https://github.com/vuejs/core-vapor/blob/877f3fcf96d96efeb51b6442c9f1c56010f5abe8/packages/compiler-vapor/__tests__/compile.test.ts#L174C1-L178C9

Reply:
It should be marked as passed, compared to vue/core. It should not be updated, the test may have to pass as expected

https://play.vuejs.org/#eNp9kU9vwjAMxb+K5Us3wUATN1TQ/ojDdtimbcdcUGq6QJpESdohVf3uc4padkDc7Pd7tp7lFh+dmzU14RLzIL1yEQLF2q2FUZWzPkILnnbQwc7bCjK2ZsIII60JEapQwirxm/tbYfL5aQPPchOpcnobiTuAvFANPEit5GElkMcmE4HrtuWq6/I507NtqYpVxiCD5s4aSeuB5/NxJ04xBs6wU+VsH6zh+G1aIFDayilN/t1FxRkFLqEniW21tr+vvRZ9TdNBlz8kDxf0fTgmTeCHp0C+IYEji1tfUjzhzdcbHbkeYWWLWrP7CvykYHWdMp5sT7UpOPY/X5/2pX+CMuV32BwjmTAclYImZ9f7BfJjnq+cfo67mC36OWE67P4A0pupHQ==

Reply:
Maybe you can change it from `test.fails` to `test`.

Reply:
You are right